config/database.yml

```diff
@@ -2,13 +2,12 @@ default: &default
   adapter: postgresql
   encoding: unicode
   database: <%= ENV['DB_NAME'] || "c2_#{Rails.env}" %>
-  pool: 5
-
-development: *default
-test: *default
-production:
-  <<: *default
   username: <%= ENV['DB_USERNAME'] %>
   password: <%= ENV['DB_PASSWORD'] %>
   host: <%= ENV['DB_HOST'] %>
   port: <%= ENV['DB_PORT'] %>
```

Comment:
If the keys are set, do we have to pass explicit defaults? For example, do we need something like this?

```erb
<%= ENV['DB_USERNAME'] || root %>
```